dev-libs/roct-thunk-interface/roct-thunk-interface-2.0.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+: ${CMAKE_MAKEFILE_GENERATOR:=ninja}
```

Comment:
this not really should be the bash variable

Reply:
Is this the only blocker before the ebuild can be submitted to the portage tree?
What are the alternatives? The eclass guide for cmake has similar solutions to set cmake related variables: https://devmanual.gentoo.org/eclass-reference/cmake-utils.eclass/index.html

Reply:
@justxi 

The devmanual is irrelevant as it is, question here if your package works with both emake and ninja or only with ninja?